### ISSUE TITLE
Summon Command?

### ISSUE THREAD

**Issue by ZackyVN** [jasonwynn10/VanillaEntityAI#4]
PMMP dont has /summon so can u make /summon command?

**Comment by jasonwynn10** [comment]
Good Idea. Ill implement that sometime with everything. for right now, the mob spawn eggs work too 👍
